src/features/user/ManageProjects/components/ProjectSites.tsx

```diff
@@ -152,7 +152,6 @@ function EditSite({
       <Fade in={openModal}>
         <form
           style={{
-            backgroundColor: 'white',
             padding: '20px',
             borderRadius: '10px',
           }}
```

Comment:
This change results in a modal with a transparent background.

![Screenshot 2025-07-04 at 2 44 22 PM](https://github.com/user-attachments/assets/d1cc24c8-56b4-47e7-b79f-c30054ba45a3)